reference-architecture/vmware-ansible/roles/openshift-auth-proxy/tasks/main.yaml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
What is the overall goal of  the auth proxy here? The diagram shows AD, wondering why LDAP isn't used until Red Hat SSO is available.